### ISSUE TITLE
Misleading Warning for Newer MySQL Versions

### ISSUE THREAD

**Issue by chrislp** [BMDan/tuning-primer.sh#25]
I am using MySQL 8.0.30 and apparently tuning-primer thinks that this is outdated and does not support query cache (which I doubt).    
If I am right and MySQL 8.0 has a query cache, the warning is wrong. If it doesn't the recommendation does not make much sense and needs to be updated.

```
QUERY CACHE
You are using MySQL 8.0.30-0ubuntu0.20.04.2, no query cache is supported.
I recommend an upgrade to MySQL 4.1 or better
```

**Comment by kshitijdeota** [comment]
query_cache has been completely removed from MySQL 8 so the warning may be right for older versions but yes, reporting needs to be updated for v8+

query_cache should be disabled anyway due to poor performance benefits. (hence feature removed in recent versions)
https://dev.mysql.com/blog-archive/mysql-8-0-retiring-support-for-the-query-cache/

**Comment by chrislp** [comment]
I see, didn't know that. Thanks for the clarification.

**Comment by BMDan** [comment]
Yeah, query_cache was always a dog on multiprocessor systems.  I'll rip this out, thanks for the suggestion.
